frameworks/cra/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@storybook/cra",
```

Comment:
do we want to call it `cra` or `create-react-app` like it was done for the preset?

Reply:
@kylegach thoughts?

Reply:
I defer to Norbert.